src/routing_membrane.rs

```diff
@@ -514,6 +518,26 @@ impl<F> RoutingMembrane<F> where F: Interface {
         }
     }
 
+    fn send_by_name<'a, I>(&self, peers: I, serialised_msg: Bytes) where I: Iterator<Item=&'a NameType> {
+        for peer in peers {
+            self.send(self.name_to_endpoint(peer).into_iter(), &serialised_msg);
+        }
+    }
+
+    fn name_to_endpoint(&self, name: &NameType) -> Vec<&Endpoint> {
+        self.relay_map.get_endpoints(&name).iter()
+            .flat_map(|&&(_, ref endpoint_set)| endpoint_set)
+            .collect()
+
```

Comment:
I think I'll need to search the `routing_table` here as well (wasn't sure when I first wrote it).